isso/db/comments.py

```diff
@@ -258,7 +258,7 @@ def fetch(self, uri, mode=5, after=0, parent='any',
         # custom sanitization
         if order_by not in ['id', 'created', 'modified', 'likes', 'dislikes', 'karma']:
             order_by = 'id'
```

Comment:
I really don't like this fallback to sort by `id` instead of `created`. Let's fix it here as well.